README.md

```diff
@@ -52,11 +100,7 @@ kubectl delete ns appmesh-system
 kubectl delete ClusterRoleBinding app-mesh-controller-binding
 kubectl delete ClusterRole app-mesh-controller
 ```
-
-**Note** that you shouldn't delete the App Mesh CRDs or the App Mesh custom resources
-(virtual nodes or services) in your cluster.
-Once you've removed the App Mesh controller and injector objects,
-you can proceed with the Helm installation as described above.
+For handling the custom resources and the CRDs please refer to either of the previous upgrade sections as relevant.
```

Comment:
Maybe it's me but I'm not sure what handling custom resources entails here. Did you mean existing custom resources?

Reply:
existing. I'll update.